Source/Core/AestraAudioController.cpp

```
@@ -242,9 +252,9 @@ bool AestraAudioController::openDefaultStream(void* userData) {
         config.deviceId = outputDevice->id;
         config.inputDeviceId = inputDevice ? inputDevice->id : outputDevice->id;
         config.sampleRate = 48000;
-        config.bufferSize = 256;
+        config.bufferSize = 512;
```

Cppcheck (2.20.0)

[style] The function 'withVibrance' is never used.

(unusedFunction)